File: test/fixtures/output/node/unirest/application-form-encoded.js
var unirest = require("unirest");

var req = unirest("POST", "http://mockbin.com/har");

req.headers({
  "content-type": "application/x-www-form-urlencoded"
});

req.form({
  "foo": "bar",
  "hello": "world",
  "hello world": "world hello"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: test/fixtures/output/python/python3/application-form-encoded.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "foo=bar&hello=world&hello%20world=world%20hello"

headers = { 'content-type': "application/x-www-form-urlencoded" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
